webdriver-spec.html

```diff
@@ -5092,12 +5092,17 @@
 
  <li><p><a>Scroll into view</a> the <var>element</var>.
 
- <li><p>Let <var>current text length</var> be the <a>Node Length</a>.
+ <li><p>If <var>element</var> is not the currently <a>active
```

Comment:
Is _currently active_ superfluous? I think “if element is not the active element” is enough.